### PR TITLE
Timesheet Calculate Time

### PR DESCRIPTION
**Pull request by rachyharkov**
Nerusin yg kalkulasi maren

### CODE REVIEW

**Comment by rachyharkov** [comment]
@Zzzul dah

**Comment by rachyharkov** [comment]
image

**Comment by rachyharkov** [comment]
image

File: resources/views/sale/time_sheet/create.blade.php [hunk 1]
```diff
@@ -296,11 +296,43 @@ function calculateTotalTimesheet() {
 
             // $('#btn-back').prop('disabled', true)
             // $('#btn-back').text('Loading...')
+            
+            // include unchecklist checkbox to be inside form data
+            let formData = new FormData()
+            $('.form-check-timesheet-input').each((i) => {
```
Comment (Zzzul) [inline resources/views/sale/time_sheet/create.blade.php:302]: mencoba code review

**Review by Zzzul — APPROVED** [review]
joss

File: resources/views/sale/time_sheet/create.blade.php (hunk 1 above)
Comment (rachyharkov) [reply resources/views/sale/time_sheet/create.blade.php:302]: kodingan ini mah aman

Comment (Zzzul) [reply resources/views/sale/time_sheet/create.blade.php:302]: wkwk iya mat, buat nambahin graph gua aja 
image

**Pull request merged**